include/rdma/providers/fi_log.h

```diff
@@ -72,7 +72,7 @@ int fi_log_ready(const struct fi_provider *prov, enum fi_log_level level,
 		 enum fi_log_subsys subsys, uint64_t *showtime);
 void fi_log(const struct fi_provider *prov, enum fi_log_level level,
 	    enum fi_log_subsys subsys, const char *func, int line,
-	    const char *fmt, ...) __attribute__ ((__format__ (__printf__, 6, 7)));
+	    const char *fmt, ...) FI_FORMAT_PRINTF(6, 7);
```

Comment:
I'm guessing that the current build always includes include/windows/osd.h prior to including this file, which is why this works today. osd.h has:
```
#define __attribute__(x)
```
Your change looks good.

Reply:
Thanks!